test/benchmark/helpers.ts

```diff
@@ -20,9 +20,26 @@ declare global {
     interface Window {
         tachometerResult: undefined | number;
         tachometerStart: undefined | 'page' | 'element';
+        tachometerEnd: undefined | 'updateComplete' | 'paint';
     }
 }
 
+/**
+ * Runs `callback` shortly after the next browser Frame is produced.
+ */
```

Comment:
If there's a URL source for this wizardry it would be good to include so we can monitor it over time.